analysis_options.yaml

```diff
@@ -1 +1,14 @@
-include: package:pedantic/analysis_options.yaml
+include: package:flutter_lints/flutter.yaml
+
+analyzer:
+  exclude: [ lib/src/api/proto/**/*.dart,
+             lib/**/*.swagger*dart]
+
+linter:
+  rules:
+    prefer_relative_imports: true
```

Comment:
Personally I am not a fan of using relative paths which is also not part of the new `flutter_lints` package requirements. 

Relative parts in small projects are okay, but when growing we experienced imports like:

```dart
import '../../../../../../widgets/foo.dart';
```

Which is impossible to scan at least for my eyes :D